yarn.lock

```diff
@@ -1630,6 +1630,24 @@
   dependencies:
     "@sinonjs/commons" "^1.7.0"
 
+"@stellar/anchor-tests@0.5.8":
```

Comment:
This seems like we're adding a dependency here. Are you sure this is necessary?

Reply:
It was added by `yarn`

Reply:
Fixed